pynecone/components/disclosure/transition.py

```diff
@@ -0,0 +1,70 @@
+"""A transition Component."""
+from typing import Union
+
+from pynecone.components.libs.chakra import ChakraComponent
+from pynecone.utils import imports
+from pynecone.vars import ImportVar, Var
+
+
+class Transition(ChakraComponent):
+    """Base componemt of all transitions."""
+
+    in_: Var[bool]
```

Comment:
Can we add comments above describing each of the props? Then they will be automatically picked up when we generate the docs

Reply:
Sure.